src/main.cpp

```diff
@@ -399,6 +399,21 @@ int main ( int argc, char** argv )
                                  argv,
                                  i,
                                  "-e",
+                                 "--directoryserver",
+                                 strArgument ) )
+        {
+            strCentralServer = strArgument;
+            qInfo() << qUtf8Printable( QString( "- directory server: %1" )
+                .arg( strCentralServer ) );
+            CommandLineOptions << "--directoryserver";
+            continue;
+        }
+
+        // Central server ** D E P R E C A T E D ** ----------------------------
```

Comment:
Should we have a `// TODO: Remove with Jamulus 4` so that we can have a quick way to grep for things to remove the next time we are allowed to do compatibility breakage?

Reply:
Mm, if we're going to do that, I'd suggest a separate issue raised to identify and tag the parts to go.  There's a lot already there.  But yes, a standardised tag would be good.